lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__miscellaneous.py

```diff
@@ -83,14 +83,16 @@ def test_stash_altname(self):
         self.check_result(cube, stashcode=self.stashcode)
 
     def test_stash_empty(self):
-        msg = "Expected STASH code MSI string"
-        with self.assertRaisesRegex(ValueError, msg):
-            self.run_testcase(ukmo__um_stash_source="")
+        value = ""
```

Comment:
Just want to check where this change originated, because I can't spot it in the head of the v3.0.x branch.

Reply:
The behaviour has now changed due to introduction of #4035, which deals with the bad `ukmo__um_stash_source` by setting it within the associated `Cube.attributes` dictionary, rather than attempting to create a `STASH` instance (which it can't because it's invalid) hence no `ValueError` is now raised.